en/docs/developer-guides/tools/migration-tools/logic-apps-migration-tool.md

```diff
@@ -0,0 +1,463 @@
+# Azure Logic Apps migration tool
```

Comment:
Add a disclaimer to the end of the page, refer to the end of https://bi.docs.wso2.com/developer-guides/tools/migration-tools/mulesoft-migration-tool/#supported-mule-versions page.

Reply:
Addressed review suggestions